grails-app/services/au/org/ala/merit/ReportService.groovy

```diff
@@ -140,7 +141,7 @@ class ReportService {
     }
 
     boolean isSubmittedOrApproved(Map report) {
```

Comment:
Probably should rename this method as it's slightly misleading now.

Reply:
I've committed the change for this.